crates/wit-parser/src/resolve.rs

```diff
@@ -600,7 +600,18 @@ impl Resolve {
                 None => {
                     log::debug!("moving interface {:?}", iface.name);
                     moved_interfaces.push(id);
-                    remap.update_interface(self, &mut iface, None)?;
+                    let pkg_id = iface.package.clone().with_context(|| {
+                        format!(
+                            "unexpectedly missing package on interface [{}] (idx: {})",
+                            iface
+                                .name
+                                .as_ref()
+                                .map(String::as_str)
+                                .unwrap_or("<unknown>"),
+                            id.index(),
+                        )
+                    })?;
```

Comment:
For these cases the `.with_context()` here and below is pretty wordy and I think it should appropriately be `.unwrap()` because a failure here isn't a user error but rather an unrecoverable error/bug in the tool itself. Given that I think it's more appropriate to panic here if the assumption is wrong rather than returning an error

Reply:
Ahhh yeah I figured it might be -- will change all these to `expect`s and let the panics flow. Thanks for laying out the rationale here

Reply:
Ah also a note -- by "and below" do you want to change *all* the `include_stability` calls as well? 

Here's an example of *just* the one being changed (which I hope is what you meant): 
https://github.com/bytecodealliance/wasm-tools/pull/1689/commits/626e77927b320cc408d16f99a61ad3c014055cc2

Reply:
Oh `include_stability` contexts are fine since those are possibly going to be shown to developers so it's definitely good to stuff more context there.